ethereum/eth/src/test/java/org/hyperledger/besu/ethereum/eth/manager/snap/SnapServerTest.java

```diff
@@ -85,7 +85,8 @@ Bytes accountRLP() {
   // force a full flat db with code stored by code hash:
   final BonsaiWorldStateKeyValueStorage inMemoryStorage =
```

Comment:
nit: Something for a next PR and not this one - I would maybe prefer to see a faked/mocked/test version of `inMemoryStorage` instead of an implementation specific one because then we are kind of being biased towards that implementation and can clearly define dependencies